test/models/content_view_test.rb

```diff
@@ -212,6 +212,40 @@ def test_all_version_library_instances_not_empty
       refute_empty @library_view.all_version_library_instances
     end
 
+    test_attributes :pid => '4a3b616d-53ab-4396-9a50-916d6c42a401'
+    def test_should_create_composite
+      view = ContentView.new(:name => "new content view", :organization_id => @organization.id)
+      [false, true].each do |composite|
+        view.composite = composite
+        assert view.valid?, "Validation failed for content view create with valid composite value: #{composite}"
+      end
+    end
+
+    test_attributes :pid => '068e3e7c-34ac-47cb-a1bb-904d12c74cc7'
+    def test_should_create_with_valid_description
+      view = ContentView.new(:name => "new content view", :organization_id => @organization.id)
+      valid_name_list.each do |description|
+        view.description = description
+        assert view.valid?, "Validation failed for content view create with valid description: '#{description}', length: #{description.length}"
+      end
+    end
+
+    test_attributes :pid => '15e6fa3a-1a65-4e7d-8d32-3a81227ac1c8'
+    def test_should_update_name_with_valid_values
+      view = ContentView.create!(:name => "org content view", :organization_id => @organization.id)
+      valid_name_list.each do |name|
+        view.name = name
+        assert view.valid?, "Validation failed for content view update with valid name: '#{name}', length: #{name}"
```

Comment:
This is tested on L66.

Reply:
@akofink  should_not allow_values this is a negative, but here we are covering a positive scenario

Reply:
If you want to test `valid_name_list` names are accepted here, then at least, let's not test them all in other places (i.e. L227).

Reply:
@akofink in L227 we are testing description field with variation values using name list

Reply:
The name and description validations in the controller have no dependencies - they're both validated individually. You should either be testing the name field validations or the description validations, not both. These are unit tests after all!